### PR TITLE
add ghactions for terraform and terrascan

### PR DESCRIPTION
**Pull request by michaelamattes**

### CODE REVIEW

File: .github/workflows/terrascan.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: terrascan
+
+on:
+  workflow_call:
+    inputs:
+      iac_type:
+        required: true
+        type: string
+        default: terraform
```
Comment (rndmh3ro) [inline .github/workflows/terrascan.yml:10]: Lets add the options here, if we want to use inputs: helm, k8s, kustomize, terraform

File: .github/workflows/terrascan.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      policy_type:
```
Comment (rndmh3ro) [inline .github/workflows/terrascan.yml:11]: Lets add the options here, if we want to use inputs: all, aws, azure, gcp, github, k8s

File: .github/workflows/terrascan.yml (hunk 1 above)
Comment (michaelamattes) [reply .github/workflows/terrascan.yml:10]: update description, no need to update worklfow if real workflow changes

File: .github/workflows/terrascan.yml (hunk 2 above)
Comment (michaelamattes) [reply .github/workflows/terrascan.yml:12]: update description, no need to update worklfow if real workflow changes

File: .github/workflows/terrascan.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+name: terrascan
+
+on:
+  workflow_call:
+    inputs:
+      iac_type:
+        required: true
+        type: string
+        default: terraform
+        description: e.g. helm, k8s, kustomize, terraform - to see all the acutally supported types look at https://github.com/tenable/terrascan-action#iac_type
```
Comment (rndmh3ro) [inline .github/workflows/terrascan.yml:11]:
```suggestion
        description: e.g. helm, k8s, kustomize, terraform - to see all supported types look at https://github.com/tenable/terrascan-action#iac_type
```

File: .github/workflows/terrascan.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      policy_type:
+        required: false
+        type: string
+        default: all
+        description: e.g. all, aws, azure, gcp, github, k8s - to see all the acutally supported types look at https://github.com/tenable/terrascan-action#policy_type
```
Comment (rndmh3ro) [inline .github/workflows/terrascan.yml:16]:
```suggestion
        description: e.g. all, aws, azure, gcp, github, k8s - to see all supported types look at https://github.com/tenable/terrascan-action#policy_type
```

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**